datasets/historical/eia_861_annual_energy_use_state_sector/dimensions/data_source_eia861.csv

```diff
@@ -0,0 +1,2 @@
+id,name
+eia861,EIA 861
```

Comment:
I think this file is no longer needed. (We changed data_source from a dimension type to a meta-data field.)